magicgui/widgets/_dialogs.py

```diff
@@ -69,6 +70,8 @@ def request_values(
         interpreted as the `annotation` in `create_widget`, by default ()
     title : str
         An optional label to put at the top., by default ""
+    parent : Widget, optional
+        An optional parent widget, by default None
```

Comment:
```suggestion
        An optional parent widget, by default None.
        The dialog will inherit style from the parent object.
```
Or something similar